code/android/Gradr/app/src/main/java/com/se2/gradr/gradr/ProfileActivity.java

```diff
@@ -321,4 +332,19 @@ private void save()
         }
 
     }
+
+    private boolean validate() {
+        boolean isValid = true;
+        String url = mPictureView.getText().toString();
+
+        if (url.length() > 0)
+            isValid = Patterns.WEB_URL.matcher(url).matches();
```

Comment:
@hnz-evn does this check if it's an imgur URL specifically? Or just that it's a URL in general?